teps/0091-trusted-resources.md

```diff
@@ -364,26 +372,161 @@ type KeylessRef struct {
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:05Z"
+          message: Trusted resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: TrustedResourcesVerified
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: All Steps have completed executing
+          reason: Succeeded
+          status: "True"
+          type: Succeeded
+      ```
+    * `trusted-resources-verification-no-match-policy` is set to `fail`,  add `false` `ConditionTrustedResourcesVerified` and `false` `Conditions.Succeeded` to `conditions`:
+      ```yaml
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:05Z"
+          message: Trusted resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: TrustedResourcesVerified
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: Succeeded
+      ```
+* Matching Policies (no matter what `trusted-resources-verification-no-match-policy` value is):
+  * all policies pass, add `true` `ConditionTrustedResourcesVerified`:
+      ```yaml
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:05Z"
+          message: Trusted resource verification passed
+          reason: ResourceVerificationSucceeded
+          status: "True"
+          type: TrustedResourcesVerified
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: All Steps have completed executing
+          reason: Succeeded
+          status: "True"
+          type: Succeeded
+      ```
+    * any enforce policy fails, add `false` `ConditionTrustedResourcesVerified` and return error in reconciler to set `Conditions.Succeeded` to `false`:
+      ```yaml
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:05Z"
+          message: Trusted resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: TrustedResourcesVerified
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: Succeeded
+      ```
+     * only warn policies fail, then only add `false` `ConditionTrustedResourcesVerified`:
+      ```yaml
+      status:
+        conditions:
+        - lastTransitionTime: "2023-03-01T18:17:05Z"
+          message: Trusted resource verification failed
+          reason: ResourceVerificationFailed
+          status: "False"
+          type: TrustedResourcesVerified
+        - lastTransitionTime: "2023-03-01T18:17:10Z"
+          message: All Steps have completed executing
+          reason: Succeeded
+          status: "True"
+          type: Succeeded
+      ```
```

Comment:
in this case, we don't actually do anything to `Conditions.Succeeded` (there could be another failure, can't assume that it'd be "True")

```suggestion
     * only warn policies fail, then only add `false` `ConditionTrustedResourcesVerified`:
      ```yaml
      status:
        conditions:
        - lastTransitionTime: "2023-03-01T18:17:05Z"
          message: Trusted resource verification failed
          reason: ResourceVerificationFailed
          status: "False"
          type: TrustedResourcesVerified
      ```
```